core/src/main/java/com/amplifyframework/auth/AuthCategoryConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth;
+
+import com.amplifyframework.core.category.CategoryConfiguration;
+import com.amplifyframework.core.category.CategoryType;
+
+/**
+ * Configuration for Auth category that also
+ * contains configuration for individual plugins.
+ */
+public final class AuthCategoryConfiguration extends CategoryConfiguration {
+    // Any category level properties would be defined here and populateFromJson would be overridden
+    // below to fill in these values from the JSON data.
+
+    /**
+     * Constructs a new AuthCategoryConfiguration.
+     */
+    public AuthCategoryConfiguration() {
+        super();
+    }
```

Comment:
This is the  behavior of the default constructor in Java, if you don't supply one 

That is to say, you could safely remove this.

Reply:
Ok, this was copying existing code from another category so I guess I can fix that across all of them.